lib/config/validation.ts

```diff
@@ -603,28 +627,13 @@ export async function validateConfig(
                   }
                 }
               }
-            } else if (
-              [
-                'customEnvVariables',
-                'migratePresets',
-                'productLinks',
-                'secrets',
-                'customizeDashboard',
-              ].includes(key)
-            ) {
-              const res = validatePlainObject(val);
-              if (res !== true) {
-                errors.push({
-                  topic: 'Configuration Error',
-                  message: `Invalid \`${currentPath}.${key}.${res}\` configuration: value is not a string`,
-                });
```

Comment:
Where/how is this validation done now? Can this be refactored in a separate PR as it seems unrelated to global config enforcement?

Reply:
This check for global options becomes obsolete cause of the above if check for global options, so I removed it.

I have separated this refactor in this PR: https://github.com/renovatebot/renovate/pull/25218

Reply:
There is no need to remove them in this PR. I will remove them in another PR.